README.md

```diff
@@ -1,3 +1,80 @@
 # Magento 1 - Url Cache Control
 
-[will follow]
\ No newline at end of file
+Magento 1.x Extension
+
+## Overview
+
+This module appends the modified timestamp to skin url's, to automatically burst Browser Cache on changes.
+
+## Requirements and setup
+
+### Requirements
+
+Tested with Magento 1.9.3.6
+
+This extension can be installed using [Composer](https://getcomposer.org/doc/01-basic-usage.md)
+
+### Setup
+
+Simple add repository to your composer.json
```

Comment:
According to our conventions, we decided we'd leave this information out because it's basically a Composer documentation which is the same for every module.

Reply:
Ok will remove that, but the Composer example is ok?

Reply:
You can leave it, sure, we have just been trying to avoid it because if we think this through, we need to clutter all readmes with general documentation for external tools. Instead, we've mostly tried to link to external documentation.

Reply:
good point. Ok it is improbably that the modul will get a different name / path, but it could happen.
maybe my usage of ionic and the dependencies npm package updates for a release version which looks like (wired collection of version numbers)[https://github.com/ionic-team/ionic/blob/master/CHANGELOG.md#370-2017-09-28] has minted me.

Personally i liked the copy & paste feature at overview.

i will remove it, You are right, we should not note to default, we should note if we have a special one